### DISCUSSION TITLE
npm install error: node gyp

### DISCUSSION THREAD

**Discussion by amarinafroes** [balta-io/forum#321] [Tira Dúvidas] [answered]
Boa noite, 
na aula  Obtendo Token do Google do curso, quando tendo executar o comando "npm install ", no terminal do fronted, da o mesmo não baixa corretamente as bibliotecas e me retorna erros:
![image](https://user-images.githubusercontent.com/89412365/235802282-558b77cd-f2dd-4aa4-9737-44600e9e53ec.png)
como conserto?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @amarinafroes, como vai?

Esse erro pelo que me lembro é por falta do Visual C++ instalado (Node GYP é um caos no Windows).

De qualquer forma, segue o link para Download do Visual C++ Build Tools:
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjGo4u-ldn-AhUTKbkGHZvNAF4QFnoECAoQAQ&url=https%3A%2F%2Fgo.microsoft.com%2Ffwlink%2F%3FLinkId%3D691126&usg=AOvVaw0geDw_h-TSCfzTMvYE2ZOw

Depois de instalar, reinicia o computador e testa novamente por favor.
